README.rst

```diff
@@ -150,6 +152,12 @@ not listed, then the accumulation of changes from previous releases is enough.
    EOX_CORE_PRE_ENROLLMENT_BACKEND: "eox_core.edxapp_wrapper.backends.pre_enrollment_l_v1"
    EOX_CORE_ENROLLMENT_BACKEND: "eox_core.edxapp_wrapper.backends.enrollment_l_v1"
 
+**Olive**
+
+.. code-block:: yaml
+
+   EOX_CORE_ENROLLMENT_BACKEND: "eox_core.edxapp_wrapper.backends.enrollment_o_v1"
```

Comment:
Maybe it's a good idea to set this setting by default; by the time the release is ready, it already has it, at least in the [common](https://github.com/eduNEXT/eox-core/blob/MJG/DS-364-olive-compatibility/eox_core/settings/common.py#L29) setting. What do you think?

Reply:
How do you usually do it? Do you update the settings according to the newer release?

Reply:
Here is an example of how we did it in [eox-hooks for nutmeg](https://github.com/eduNEXT/eox-hooks/pull/33/files#diff-1e59edcaa6ca0f458267b12ee83e2d023a4453073ba21db192904c88d918b8e5).

Reply:
I couldn't find any nutmeg backend in the files you linked. Do you usually change the default common settings according to the newer supported release? ie, the enrollment backend will now be the olive one as default. I don't mind changing it, but I want to be consistent.

Reply:
Sorry, I thought we changed the backend. If we release the olive version, we can have that setting by default, but you are right in thinking about consistency. I really don't know. If you think it's better no change it for consistency I'm okay, maybe we can start a discussion about this later.